runner/Makefile

```diff
@@ -1,9 +1,10 @@
-NAME ?= summerwind/actions-runner
-DIND_RUNNER_NAME ?= ${NAME}-dind
+DOCKER_USER ?= summerwind
+RUNNER_NAME ?= ${DOCKER_USER}/actions-runner
+DIND_RUNNER_NAME ?= ${DOCKER_USER}/actions-runner-dind
 TAG ?= latest
-TARGET_PLATFORM ?= $(shell arch)
+TARGETPLATFORM ?= $(shell arch)
```

Comment:
Would've been better to move from `TARGETPLATFORM` -> `TARGET_PLATFORM` than the inverse...

Reply:
Docker convetion is upper case no spaces so I followed that tbh e.g. `TARGETPLATFORM`, `BUILDPLATFORM`, `TARGETVARIANT` etc etc